netbout-web/src/test/java/com/netbout/dynamo/DyMessagesITCase.java

```diff
@@ -153,4 +159,31 @@ public void codeMarkdownInMessages() throws Exception {
             Matchers.equalTo("    4 leading spaces retained")
         );
     }
+    
+    /**
+     * DyMessages has to throw {@code HttpException} if invoke {@code unread()}
+     * for user which is not in a bout.
+     * @throws URISyntaxException If there is some problem inside
+     * @throws IOException If there is some problem inside
+     */
+    @Test(expected = HttpException.class)
+    public void exceptionIfBoutNotFound() throws IOException {
+        Messages messages = null;
+        try {
+            final String name = "testunread";
+            final Aliases aliases =
+                new DyBase().user(new URN("urn:test:75066")).aliases();
+            final Alias alias = aliases.add(name);
+            final Inbox inbox = alias.inbox();
+            final Bout bout = inbox.bout(inbox.start());
+            bout.messages().post("    4 leading spaces retained  ");
+            final Friends friends = bout.friends();
+            friends.kick(name);
+            messages = bout.messages();
+        } catch (IOException | URISyntaxException e) {
+            fail(e.getMessage());
```

Comment:
@2686747 why do we need this block in the first place?
We're trying to trigger `HttpException`, why do we need a `try` block around that to begin with ?